logging/logmanager/plugin_impl_log_manager.go

```diff
@@ -216,3 +234,91 @@ func stringToLogLevel(level string) logging.LogLevel {
 
 	return logging.InfoLevel
 }
+
+// list of supported hook services to send errors to remote syslog server
+const (
+	HookSysLog   = "syslog"
+	HookLogStash = "logstash"
+	HookFluent   = "fluent"
+)
+
+// HookConfig contains configuration of hook services
+type HookConfig struct {
+	Protocol string
+	Address  string
+	Port     int
+	Levels   []string
+}
+
+// CommonHook implements that Hook with own level definition
+type CommonHook struct {
```

Comment:
No need to export this type, it's not gonna be useful for anyone outside this package.